src/displayapp/screens/PineTimeStyle.cpp

```diff
@@ -235,6 +233,15 @@ void PineTimeStyle::Refresh() {
     lv_obj_realign(notificationIcon);
   }
 
+  if (notificationState.Get() && bleState.Get()) {
+    lv_obj_align(bleIcon, sidebar, LV_ALIGN_IN_TOP_MID, 8, 25);
+    lv_obj_align(notificationIcon, sidebar, LV_ALIGN_IN_TOP_MID, -8, 25);
+  } else if (notificationState.Get() && !bleState.Get()) {
+    lv_obj_align(notificationIcon, sidebar, LV_ALIGN_IN_TOP_MID, 0, 25);
+  } else {
+    lv_obj_align(bleIcon, sidebar, LV_ALIGN_IN_TOP_MID, 0, 25);
+  }
+
```

Comment:
> Don't we want to keep the icons in a straight line? I think that would make the most sense for a vertical bar.

I prefer it visually having some whitespace either side of the calendar icon, but it is a very minor preference given the notification icon and bluetooth icon are almost never displayed at the same time. However, it's also a point of pride at this point since this setup was the original goal so I want to make it work goddamnit!
Also, my next goal is going to be integrating weather into the watchface which is probably going to mean moving these icons again
I'll give your suggestion a try for using IsUpdated!